### PR TITLE
Add documentation for MELCloud integration

### PR DESCRIPTION
**Pull request by vilppuvuorinen**
## Proposed change

This PR adds documentation for a new integration to MELCloud.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: home-assistant/home-assistant#30712
- This PR fixes or closes issue:

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/melcloud.markdown [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: MELCloud
+description: MELCloud integration
+logo: home-assistant.png
+ha_category:
+  - Climate
+ha_release: '0.106'
```
Comment (klaasnicolaas) [inline source/_integrations/melcloud.markdown]:
```suggestion
ha_release: 0.106
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by rudizl** [comment]
Strange, it looks like the system didn't see that the requested change is done. In To Do list too...

**Comment by vilppuvuorinen** [comment]
I didn't realize I could just commit the change here and then it was already out dated. I'd rate this as not completely my fault.

File: source/_integrations/melcloud.markdown [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+ha_category:
+  - Climate
+ha_release: 0.106
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@vilppuvuorinen'
+---
+
+The `melcloud` integration integrates Mitsubishi Electric's MELCloud enabled devices into Home Assistant.
```
Comment (rytilahti) [inline source/_integrations/melcloud.markdown]: Is there some site that could be linked here?

File: source/_integrations/melcloud.markdown [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+## Device support
+
+- Air-to-Air heat pumps, i.e. AC units - **Supported**
```
Comment (rytilahti) [inline source/_integrations/melcloud.markdown]: do you mean "e.g." instead of "i.e." here?

File: source/_integrations/melcloud.markdown [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+- Air-to-Water heat pumps - **Work in progress**
+- Energy recovery ventilators - **Not supported**
+- Other - **Not supported**
+
+## Configuration
+
+MELCloud integration should be configured through the Home Assistant user interface using MELCloud  login details. This is not optimal, but **the provided password is not stored**.
```
Comment (rytilahti) [inline source/_integrations/melcloud.markdown]:
```suggestion
The integration should be configured through the user interface ("Configurations -> Integrations") using the MELCloud login details. While not optimal, **the provided password is not stored**.
```

File: source/_integrations/melcloud.markdown [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+The stored access token can later be updated by adding the MELCloud integration again with the same email address.
```
Comment (rytilahti) [inline source/_integrations/melcloud.markdown]: Can or need to be updated?

File: source/_integrations/melcloud.markdown [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+Configuration is also possible through `configuration.yaml`. The token can be found from `X-MitsContextKey` header when logged into MELCloud with 'English' and 'Remember me'.
```
Comment (rytilahti) [inline source/_integrations/melcloud.markdown]:
```suggestion
Configuration is also possible through `configuration.yaml`. The required authentication token can be found in `X-MitsContextKey` header when logged into the MELCloud with 'English' and 'Remember me'.
```

Comment (rytilahti) [reply source/_integrations/melcloud.markdown]: "with 'English' and 'Remember me'" sounds odd. Is there are word missing or what's the message this is trying to convey? That it only works when the language is set to English and the "Remember me" option is selected while logging in?

File: source/_integrations/melcloud.markdown [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+Following parameters can be controlled throught the `climate` platform:
+
+- Target temperature
+- Operation mode (HVAC mode) with emulated power off.
+- Fan speed
+
+### Sensor
+
+Following parameters are provided through `sensor` platform:
```
Comment (rytilahti) [inline source/_integrations/melcloud.markdown]:
```suggestion
The following attributes are available for `sensor` platform entities:
```

File: source/_integrations/melcloud.markdown [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+  type: string
+{% endconfiguration %}
+
+## Air-to-Air device
+
+Air-to-Air heat pump provides `climate` and `sensor` platforms.
+
+### Climate
+
+Following parameters can be controlled throught the `climate` platform:
```
Comment (rytilahti) [inline source/_integrations/melcloud.markdown]:
```suggestion
The following parameters can be controlled for the `climate` platform entities:
```

File: source/_integrations/melcloud.markdown [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+
+- Target temperature
+- Operation mode (HVAC mode) with emulated power off.
```
Comment (rytilahti) [inline source/_integrations/melcloud.markdown]: Is "emulated power off" understandable for end users?

File: source/_integrations/melcloud.markdown (hunk 2 above)
Comment (vilppuvuorinen) [reply source/_integrations/melcloud.markdown]: Linked.

File: source/_integrations/melcloud.markdown (hunk 3 above)
Comment (vilppuvuorinen) [reply source/_integrations/melcloud.markdown]: Not really, but it does look a lot better with "e.g.". Switched.

File: source/_integrations/melcloud.markdown (hunk 9 above)
Comment (vilppuvuorinen) [reply source/_integrations/melcloud.markdown]: You'll never know. I tried to clarify this a bit.

File: source/_integrations/melcloud.markdown (hunk 6 above)
Comment (vilppuvuorinen) [reply source/_integrations/melcloud.markdown]: I repurposed the latter sentence.

File: source/_integrations/melcloud.markdown (hunk 5 above)
Comment (vilppuvuorinen) [reply source/_integrations/melcloud.markdown]: Attempted to clarify.

**Review by frenck — APPROVED** [review]

**Pull request merged**